code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -433,12 +433,15 @@ func (ssc *StorageSmartContract) GetReadMarkersHandler(ctx context.Context,
 		limit = l
 	}
 
-	if isDescendingString != "" {
-		isD, err := strconv.ParseBool(isDescendingString)
-		if err != nil {
-			return nil, errors.New("is_descending is invalid")
+	if sortString != "" {
+		switch sortString {
+		case "desc":
+			isDescending = true
+		case "asc":
+			isDescending = false
+		default:
+			return nil, errors.New("sort value is invalid")
 		}
-		isDescending = isD
 	}
 
 	readMarkers, err := balances.GetEventDB().GetReadMarkersFromQueryPaginated(query, offset, limit, isDescending)
```

Comment:
what is isDescending?
I don't see sortString being used..

Reply:
```
	if sortString != "" {
		switch sortString {
		case "desc":
			isDescending = true
		case "asc":
			isDescending = false
		default:
			return nil, errors.New("sort value is invalid")
		}
	}
```
I had used this... but maybe you missed this

Reply:
The reason being we have to validate the parameters. So if we already evaluating the parameter why not convert it to boolean instead of calculating it again?

Reply:
it's ok. api should not be exact equal to code